src/sentry/net/http.py

```diff
@@ -160,7 +160,6 @@ def init_poolmanager(self, connections, maxsize, block=DEFAULT_POOLBLOCK, **pool
             num_pools=connections,
             maxsize=maxsize,
             block=block,
-            strict=True,
```

Comment:
?

Reply:
the parameter is gone -- it hasn't done anything since we upgraded to python 3.x anyway